packages/contracts/contracts/FeaturedVotingContract.sol

```diff
@@ -182,7 +182,7 @@ contract FeaturedVotingContract {
         return a < b ? a : b;
     }
 
-    function _isInCooldownPeriod(bytes calldata publicKey) private view returns (bool) {
+    function isInCooldownPeriod(bytes calldata publicKey) public view returns (bool) {
```

Comment:
For completeness-sake we could add a test for this function, now that it's user-facing.

Reply:
:+1: [done](https://github.com/status-im/community-dapp/pull/39/files#diff-6110042bee76cd819a2aefbb013265951aab62244de9373a1e399ab8253418dd)

I decided to remove one test because it started to fail after this change:
```diff
--- a/packages/contracts/test/3.featuredVotingContract.test.ts
+++ b/packages/contracts/test/3.featuredVotingContract.test.ts
@@ -22,7 +22,7 @@ const publicKeys = [
 const votingLength = 1000
 const votingVerificationLength = 200
 const featuredPerVotingCount = 3
-const cooldownPeriod = 1
+const cooldownPeriod = 2
 const votingWithVerificationLength = votingLength + votingVerificationLength
 
 const typedData = {
```

The same code path is now covered in `#isInCooldownPeriod`.

Reply:
Nice, this works as well.
Thank you